data/vocab/scholar.ttl

```diff
@@ -866,80 +429,185 @@ scholar:suffix a rdf:Property ;
+    rdfs:range bf:Concept ;
+    rdfs:domain bf:Person .
+
+scholar:discipline a rdf:Property ;
+    rdfs:label "discipline"@en ;
+    rdfs:comment "A field of study or academic discipline related to a person."@en ;
+    rdfs:range bf:Concept ;
+    rdfs:domain bf:Person .
+
 ## Crossref ID properties
+
 scholar:fainId a rdf:Property ;
-    rdfs:label "FAIN ID"@en ;
+    rdfs:label "fain ID"@en ;
     rdfs:comment "A unique identifier for a funding award."@en ;
     rdfs:domain scholar:FundingAward ;
     rdfs:range rdfs:Literal .
 
 scholar:ebscoId a rdf:Property ;
-	rdfs:label "EBSCO ID"@en ;
+	rdfs:label "ebsco ID"@en ;
 	rdfs:comment "A unique identifier for a EBSCO linked data resources."@en ;
 	rdfs:domain scholar:FundingAward ;
+	rdfs:range rdfs:Literal .
+	
+scholar:doi a rdf:Property ;
+	rdfs:label "doi"@en ;
+	rdfs:comment "A persistent identifier or handle used to uniquely identify various objects, standardized by the International Organization for Standardization."@en ;
+	rdfs:domain scholar:Article ;
 	rdfs:range rdfs:Literal ;
-    rdfs:domain [
-        a owl:Class ;
-        owl:unionOf (bf:Article bf:Person bf:Organization)
-    ] .
-
+    rdfs:range rdfs:Literal .
+	
 ## Crossref property for Award
+
 scholar:award a rdf:Property ;
     rdfs:label "award"@en ;
     rdfs:comment "An award associated with a person, organization, or resource."@en ;
     owl:equivalentProperty schema:award ;
     rdfs:domain bf:Agent ;
     rdfs:range scholar:Award .
 
-## Wikidata properties for Person (added for National Library of Estonia)
+## Scholar properties for Work, Article, and Award
 
-bf:hasOccupation a rdf:Property ;
-    rdfs:label "occupation"@en ;
-    rdfs:comment "The main vocation, job, or employment activity of an individual."@en ;
-    owl:equivalentProperty schema:hasOccupation ;
-    rdfs:domain bf:Person ;
-    rdfs:range rdfs:Literal  .
+scholar:cites a rdf:Property ;
+    rdfs:label "cites"@en ;
+    rdfs:comment "Links the citing work to the cited work."@en ;
+    rdfs:domain bf:Work ;
+    rdfs:range bf:Work .
 
-bf:education a rdf:Property ;
-    rdfs:label "education"@en ;
-    rdfs:comment "Educational background or the highest level of formal education attained by an individual"@en ;
-    rdfs:domain bf:Person ;
-    rdfs:range rdfs:Literal  .
+scholar:citedBy a rdf:Property ;
+    rdfs:label "citedBy"@en ;
+    rdfs:comment "Links cited work to the citing work."@en ;
+    rdfs:domain bf:Work ;
+    rdfs:range bf:Work .
+
+scholar:citedByCount a rdf:Property ;
+    rdfs:label "cited by count"@en ;
+    rdfs:comment "The number of times a Work has been cited."@en ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bf:Work .
+
+scholar:citationCount a rdf:Property ;
+    rdfs:label "citation count"@en ;
+    rdfs:comment "The number of citations used by a Work."@en ;
+    rdfs:range rdfs:Literal ;
+    rdfs:domain bf:Work .
+
+scholar:articleVersion a rdf:Property ;
```

Comment:
```suggestion
scholar:instances a rdf:Property ;
	rdfs:label "instances"@en ;
	rdfs:comment "One or more instances of the work."@en ;
	rdfs:domain bld:Work ;
	rdfs:range bld:Instance .

scholar:articleVersion a rdf:Property ;
```
Do we need this field? It would make the API compliant with the vocabulary.